cgcnn2/cli/cgcnn_ft.py

```
@@ -29,6 +28,191 @@
+        ),
+    )
+    parser.add_argument(
+        "-al",
+        "--axis-limits",
+        nargs=2,
+        default=None,
+        type=float,
+        help="The limits for the x and y axes of the parity plot.",
+    )
+
+    args = parser.parse_args(sys.argv[1:])
```

Ruff (0.8.2)

Undefined name `sys`

(F821)